packages/forms/src/UIForm/fields/FieldTemplate/FieldTemplate.scss

```diff
@@ -2,4 +2,8 @@
 	&:global(.required) :global(.control-label):after {
 		content: '*';
 	}
+	&:global(.required) :global(.checkbox) :global(.control-label):after {
```

Comment:
```suggestion
.template {
	:global {
		.required .checkbox .control-label:after {
			position: initial;
			content: '*';
			outline: none;
		}
	}
}
```

Reply:
`position: initial` is not compatible with IE 11
https://caniuse.com/#feat=css-initial-value